emspy/query/flight.py

```diff
@@ -75,13 +76,19 @@ def __save_fieldtree(self):
 
 
     def __get_dbtree(self):
-        T = self._metadata.get_data("dbtree", "ems_id = %d" % self._ems_id)
+        filter = "ems_id = %d" % self._ems_id
```

Comment:
Should this be an attribute of the object, since the same code gets repeated multiple times throughout the class?  Maybe there should be a __set_filter() and __get_filter() methods?

Reply:
Yeah I can look into making this an attribute